src/main/java/yevhen/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package yevhen.bookstore.dto;
+
+import java.math.BigDecimal;
+
+public record CreateBookRequestDto(
+        Long id,
+        String title,
+        String author,
+        String isbn,
+        BigDecimal price,
+        String description,
+        String coverImage
+) {}
```

Comment:
You shouldn't use id in this DTO
```suggestion
public record CreateBookRequestDto(
        String title,
        String author,
        String isbn,
        BigDecimal price,
        String description,
        String coverImage
) {}
```